modin/pandas/test/dataframe/test_reduction.py

```diff
@@ -375,3 +353,16 @@ def test_sum_single_column(data):
     pandas_df = pandas.DataFrame(data).iloc[:, [0]]
     df_equals(modin_df.sum(), pandas_df.sum())
     df_equals(modin_df.sum(axis=1), pandas_df.sum(axis=1))
+
+
+@pytest.mark.parametrize(
+    "fn", ["max", "min", "median", "skew", "kurt", "sem", "std", "var"]
```

Comment:
This list is not the same as the list of affected operations.

Reply:
the `mean` method was missing, added it